scalekit/core.py

```
@@ -254,14 +309,14 @@ def grpc_exec(
                     raise ScalekitServerException.promote(exp)
                 try:
                     self.__authenticate_client()
-                    return self.grpc_exec(func, data, retry=retry-1)
+                    return self.grpc_exec(func, data, retry=retry-1, timeout=timeout)
                 except Exception as refresh_exp:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)